ui/components/app/snaps/snap-ui-address/__snapshots__/snap-ui-address.test.tsx.snap

```diff
@@ -172,7 +172,7 @@ exports[`SnapUIAddress renders Ethereum address 1`] = `
           <rect
             fill="#F29602"
             height="32"
-            transform="translate(-2.81750896228304 4.951916544006229) rotate(230.4 16 16)"
+            transform="translate(-2.8175089622830405 4.951916544006229) rotate(230.4 16 16)"
```

Comment:
i assume the changes in these 2 other files aren't needed?

Reply:
I'm not sure why these snapshots are updating actually. Trying to figure that out now. Getting some unexpected errors here

Reply:
I do see that the icons in AvatarNetwork recently updated. I wonder if that is somehow related